docs/installation/_index.md

```diff
@@ -111,47 +98,30 @@ To install `viam-server` on a macOS computer:
 
    Click **View setup instructions** to open the setup instructions.
 
-1. Select **Mac** as your system's OS and **RDK** as your RDK type.
-
-1. Follow the steps shown to install `viam-server` on your macOS computer.
-
-1. Once you have followed the steps on the setup instructions, `viam-server` is installed and running.
-   Wait for confirmation that your computer has successfully connected.
-
-{{% /tab %}}
-{{% tab name="Windows" %}}
-
-1. Go to the [Viam app](https://app.viam.com).
-   Create an account if you haven't already.
-
-1. Add a new machine by providing a name in the **New machine** field and clicking **Add machine**:
-
-   ![The 'First Location' page on the Viam app with a new machine name in the New machine field and the Add machine button next to the field highlighted.](/fleet/app-usage/create-machine.png)
-
-1. Navigate to the **CONFIGURE** tab and find your machine's card.
-   An alert will be present directing you to **Set up your machine part**:
-
-   ![Machine setup alert in a newly-created machine](/installation/setup-part.png)
+1. Select your system's operating system and **RDK** as your RDK type.
 
-   Click **View setup instructions** to open the setup instructions:
+1. Select your preferred installation method:
 
-   ![Setup instructions](/installation/wsl-setup-instructions.png)
+   - `manual`: installs only `viam-server` on your machine.
+   - `viam-agent`: installs viam-agent, which will automatically install (or update) viam-server **and** provide additional functionality such as [provisioning](/fleet/provision/) and operating system update configuration.
+   - installs [`viam-agent`](/configure/agent/) which provisions a machine as it first comes online with a pre-defined configuration.
 
-1. Select **Windows** as your system's OS and **RDK** as your RDK type.
+1. Follow the steps shown on your machine.
 
-1. Follow the steps shown to install `viam-server` on your Windows machine.
+1. Once you have followed the steps on the setup instructions, wait for confirmation that your machine has successfully connected.
 
-1. Once you have followed the steps on the setup instructions, `viam-server` is installed and running.
-   Wait for confirmation that your computer has successfully connected.
+   On your machine's page on [the Viam app](https://app.viam.com), your machine will show that it's **Live**.
 
-{{% /tab %}}
-{{< /tabs >}}
+{{< alert title="Linux: Automatic startup" color="note" >}}
+On manual Linux installs, `viam-server` will start automatically when your system boots.
+You can [change this behavior](/installation/manage-viam-server/) if desired.
+{{< /alert >}}
```

Comment:
This happens for agent too, just how you manage it is slightly different (e.g. it's the viam-agent service, instead of the viam-server service.)